server/client.go

```diff
@@ -302,8 +302,7 @@ type outbound struct {
 	stc chan struct{} // Stall chan we create to slow down producers on overrun, e.g. fan-in.
 }
 
-const nbPoolSizeSmall = 512   // Underlying array size of small buffer
```

Comment:
Still think this might have been a good idea.

Reply:
Could you test with small payloads, 32, 64 or 128 bytes and watch RSS?

Reply:
Interestingly, on NATS Core publishes, there seems to be virtually no difference in how two vs three buffers manifest, but in JetStream publishes, some of the benchmarks show better results. I've added the small buffer back as a result.